cluster-autoscaler/cloudprovider/azure/test/e2e/azure_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build e2e
```

Comment:
This build tag ensures that the `go test` invocation on release branches that use `GO111MODULE=off` doesn't catch Azure e2e packages since they will not contain unit tests.